### DISCUSSION TITLE
Welcome to WesternMeteorPyLib Discussions!

### DISCUSSION THREAD

**Discussion by dvida** [wmpg/WesternMeteorPyLib#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by markmac99** [comment]
I'm mark mcintyre. I'm based in the UK. I did my physics degree in the 1980s, but after a doctorate in engineering i went out into the wide world and worked in financial services for a few decades, before coming back and doing a second degree, this time focusing on astrophysics. I'm a member of the IOP, BAA, BIS and meteor section director for the UK's Society for Popular Astronomy. 
I'm a contributor to the UK Meteor Network (UKMON) and to the Global Meteor Network. In UKMON we've been using the UFO suite but are keen to move away from it as it is cumbersome, and we really don't know what it is doing internally. We also want to improve & automate data analysis and processing, leveraging the power of cloud computing. So I'm using WMPL to perform matching and trajectory analysis on our data which is on AWS,. The data includes contributions from FRIPON and UKMON as well as a few RMS cameras.
